tmt/tests/bootc-install-provision.sh

```diff
@@ -19,7 +19,27 @@ case "$ID" in
 esac
 
 if [ "$TMT_REBOOT_COUNT" -eq 0 ]; then
+    # Fedora CI: https://github.com/fedora-ci/dist-git-pipeline/blob/master/Jenkinsfile#L145
+    # OSCI: https://gitlab.cee.redhat.com/osci-pipelines/dist-git-pipeline/-/blob/master/Jenkinsfile?ref_type=heads#L93
+    if [[ -v KOJI_TASK_ID ]] || [[ -v CI_KOJI_TASK_ID ]]; then
+        echo "$TMT_SOURCE_DIR"
+        ls -al "$TMT_SOURCE_DIR"
+        ls -al "$TMT_SOURCE_DIR/SRPMS"
+        ls -al /etc/yum.repos.d
+        cat /etc/yum.repos.d/test-artifacts.repo
+        ls -al /var/share/test-artifacts
+        # get bootc and system-reinstall-bootc rpm package name
+        BOOTC_RPM_FULL_PATH=$(find /var/share/test-artifacts/ -type f | grep -E '/bootc-[0-9]{1,2}.[0-9]{1,2}.[0-9]{1,2}-[0-9]{1,2}.(fc|el)[0-9]{1,2}.(x86_64|aarch64).rpm$')
```

Comment:
Why can't we just `find /var/share/test-artifacts -type f -name '*.rpm' -exec cp {} "$BOOTC_TEMPDIR" \;` ?

Reply:
There're `src` and `debug` rpms in the same folder. So I have to filter two rpms with regex.

Reply:
OK. At least if we do the bind mount we won't need to filter here. And I think `rpm -Fvh --oldpackage` should ensure we only upgrade already installed packages.

Reply:
I removed all those regrex and use bind mount and `dnf -y upgrade /rpms/*.rpm` instead.